cachito/web/migrations/versions/7d979987402d_delete_dependencies_and_packages.py

```diff
@@ -0,0 +1,135 @@
+"""Delete dependencies and packages
+
+Revision ID: 7d979987402d
+Revises: 976b7ef3ec86
+Create Date: 2021-10-12 15:23:03.826535
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+from alembic import context
+
+# revision identifiers, used by Alembic.
+revision = "7d979987402d"
+down_revision = "976b7ef3ec86"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    if context.get_x_argument(as_dictionary=True).get("delete_data", None):
+        with op.batch_alter_table("request_dependency", schema=None) as batch_op:
+            batch_op.drop_index("ix_request_dependency_dependency_id")
+            batch_op.drop_index("ix_request_dependency_package_id")
+            batch_op.drop_index("ix_request_dependency_replaced_dependency_id")
+            batch_op.drop_index("ix_request_dependency_request_id")
+
+        op.drop_table("request_dependency")
+        with op.batch_alter_table("request_package", schema=None) as batch_op:
+            batch_op.drop_index("ix_request_package_package_id")
+            batch_op.drop_index("ix_request_package_request_id")
+
+        op.drop_table("request_package")
+        with op.batch_alter_table("package", schema=None) as batch_op:
+            batch_op.drop_index("ix_package_dev")
+            batch_op.drop_index("ix_package_name")
+            batch_op.drop_index("ix_package_type")
+            batch_op.drop_index("ix_package_version")
+
+        op.drop_table("package")
+
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    if context.get_x_argument(as_dictionary=True).get("delete_data", None):
```

Comment:
do we need it in downgrade?

Reply:
Okay, reverted back :D No we can merge that :D